target/linux/ramips/dts/mt7628an_zyxel_keenetic-4g.dts

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "zyxel,keenetic-4g", "mediatek,mt7628an-soc";
+	model = "Keenetic 4G";
+
+	aliases {
+		led-boot = "/leds/power";
+		led-failsafe = "/leds/power";
+		led-running = "/leds/power";
+		led-upgrade = "/leds/power";
```

Comment:
You do not define a power-led, and the definition here would be different anyway.